### ISSUE TITLE
/opt/ttn-gateway/packet_forwarder compile failure: missing directory

### ISSUE THREAD

**Issue by hdtodd** [RAKWireless/rak_common_for_gateway#30]
It appears that the install process was modified so that the source code copied into /opt/ttn-gateway during the install process does not include /opt/ttn-gateway/lora_gateway.  As a result, attempts to "cd /opt/ttn-gateway/packet_forwarder/lora_pkt_fwd; make clean; make" fail because the files needed in /opt/ttn-gateway/lora_gateway aren't there.  It looks like the "cp" that should be there in the "install.sh" file has been misplaced and is avoided, as the "else" clause of an "if".  

This can be fixed manually once you know where to look, but it would be very helpful if the install.sh script were modified to copy BOTH the packet_forwarder and lora_gateway directories to /opt/ttn-gateway during the install process.

**Comment by RAKzhuqi** [comment]
Hello @hdtodd ，
Do you clone the source code directly or download the compressed package?
I cloned the code and tested rak2245, it is ok.

**Comment by hdtodd** [comment]
@RAKzhuqi,

I cloned it to install.  I've done it a number of times and tried to clean up each time so that each new install was clean.  

For my first few installs, several months ago, I installed chirpstack.  But I really just want the ttn-gateway, so I removed the /opt/ttn-gateway directory and subdirectories and then did a fresh install with --chirpstack=not_install.  That's when I think the lora_gateway subdirectory didn't get copied over to /opt/ttn-gateway.  I knew where to look, so I copied it over manually, and I can recompile the the Makefile from /opt/ttn-gateway/packet_forwarder/lora_pkt_fwd with no problems.  It just seemed that the /opt/ttn-gateway/lora_gateway directory didn't get copied as part of the install procedure with --chirpstack=not_install.

If I need to do another install, I'll take more careful notes.  But I think the problem occurs when you don't install chirpstack.

David
